staff/alexadark/task-api/src/index-1.0.0.js

```diff
@@ -0,0 +1,135 @@
+require('dotenv').config()
+const express = require('express')
+const bodyParser = require('body-parser')
+const _ = require('lodash')
+const app = express()
+const router = express.Router()
+app.use('/api', router)
+const port = process.env.PORT
+const jsonBodyParser = bodyParser.json()
+
+const tasks = []
+
+/**
+ * List todoList
+ */
+router.get('/tasks/todo', (req, res) => res.json(success('tasks successfully listed', tasks.filter(task => !task.done))))
+
+/**
+ * Create task
+ */
+router.post('/tasks', jsonBodyParser, (req, res) => {
+    const {id, text} = req.body
+
+    if (!id || !text) res.json(fail('tasks creation failed', 'invalid id or text'))
+
+
+    if (taskIndex(tasks, id) > -1) res.json(fail('task creation failed', 'task already exists'))
+
+    tasks.push({id, text})
+
+    res.json(success('task added', req.body))
+})
+
+/**
+ * Delete task
+ */
+router.delete('/tasks/:id', jsonBodyParser, (req, res) => {
+    const idParam = req.params.id
```

Comment:
RENAME idParam to just id, no need to add suffix -Param, it is already clear to be a parameter (AVOID unnecessary semantics).